README.md

```diff
@@ -1,49 +1,60 @@
-jbi.vim
-=======
+## motoman-inform-vim-plugin
 
-Vim syntax file for Motoman INFORM III.
-----------------------------------------------------------------------
+## [Vim][1] syntax and filetype plugin for [Motoman][2] INFORM III.
 
 ### Installation
 
-Copy jbi.vim to `~/.vim/syntax/` or `$VIMRUNTIME/syntax/`.
-Use the command
+#### Installation with vim-plug:
 
-```vim
-    :set ft=jbi
-```
+Put this in your .vimrc:
 
-to enable syntax highlighting for jbi files.
+    call plug#begin('~/.vim/plugged')
```

Comment:
Consider using `` ```vim `` and `` ``` `` to surround this (and the other) example code blocks. Github will then highlight these snippets.

Reply:
done.